code/ZAS/XGM_reactions.dm

```diff
@@ -27,19 +27,20 @@
 	name = "Cryotheum-Oxygen Reaction"
 
 /datum/gas_reaction/cryotheum_oxygen_reaction/reaction_is_possible(datum/gas_mixture/mixture)
-	return mixture[GAS_CRYOTHEUM] > 0 && mixture[GAS_OXYGEN] > 0
+	return mixture[GAS_CRYOTHEUM] > 0 && mixture.molar_density(GAS_CRYOTHEUM) * CELL_VOLUME > MOLES_CRYOTHEUM_VISIBLE && mixture[GAS_OXYGEN] > 0
 
 /datum/gas_reaction/cryotheum_oxygen_reaction/reaction_amounts_requested(datum/gas_mixture/mixture)
 	var/to_return[] = list()
+	// Reach maximum efficiency when the Cryo:Oxy ratio is 1:50 or higher. Any less and scale linearly, i.e. 1:100 is half speed.
 	var/catalyst_coefficient = min(1.0, mixture[GAS_CRYOTHEUM] / mixture[GAS_OXYGEN] * 50)
 	to_return[GAS_OXYGEN] = catalyst_coefficient * mixture[GAS_OXYGEN]*0.0015
 	return to_return
 
 /datum/gas_reaction/cryotheum_oxygen_reaction/perform_reaction(datum/gas_mixture/mixture, reactant_amounts)
 	var/reaction_coefficient = reactant_amounts[GAS_OXYGEN]
 	mixture[GAS_OXYGEN] = max(0, mixture[GAS_OXYGEN] - reaction_coefficient)
-	// -120000 times 0.0015 * mols of oxygen should reduce a normal room by about 1.7C every time this ticks.
-	mixture.add_thermal_energy( reaction_coefficient * -120000, 232.8952)
+	// Should reduce a normal room down to a minimum of around -30C in less than a minute.
+	mixture.add_thermal_energy( reaction_coefficient * -170000, 242.8952)
```

Comment:
Make this easier to tweak in the future by killing the magic numbers either via new vars/defines or use existing ones here

Reply:
I'll make it a variable at least.